aircan/dags/api_ckan_import_to_bq_v2.py

```
@@ -0,0 +1,103 @@
+        }
+        aircan_status_update(ckan_site_url, ckan_api_key, status_dict)
+    except Exception as e:
+        status_dict = {
+        'dag_run_id': dag_run_id,
+        'resource_id': res_id,
+        'state': 'failed',
+        'message': str(e),
+        'clear_logs': True
+        }
+        aircan_status_update(ckan_site_url, ckan_api_key, status_dict)
+        raise Exception(str(e))
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)